src/helpers/getResultStringForAdditionalPropertiesType.ts

```diff
@@ -38,5 +36,16 @@ export const getResultStringForAdditionalPropertiesType = ({
             break;
     }
 
-    return `${getDescription(description)}\t${propertyName}: {\n [key: string]: ${res};\n };\n`;
+    if (xDictionaryKey && !!xDictionaryKey[SwaggerProps.$ref]) {
```

Comment:
this file was already added by https://github.com/LandrAudio/openapi-codegen-typescript/pull/52 but instead of having two branch for additional properties with and without dictionnary - and duplicate some logic, I added the check here

Reply:
you mean changing the base to the PR #52 ? It's already the case.